src/server/logger.js

```diff
@@ -34,6 +34,7 @@ log.levelEnums = {
   SILENT: 5,
 };
 
+log.setLevel('INFO');
```

Comment:
why is this line necessary?

Reply:
seems the leveled log has default level and won't output my first INFO log in config.js... so I just hard set the level here